test/emqx_node_helpers.erl

```diff
@@ -70,10 +91,10 @@ setup_node(Node, #{} = Opts) ->
         end,
     EnvHandler = maps:get(env_handler, Opts, DefaultEnvHandler),
 
-    [ok = rpc:call(Node, application, load, [App]) || App <- [gen_rpc, emqx]],
+    [ok = rpc:call(Node, application, load, [App]) || App <- [gen_rpc, emqx_modules, emqx]],
     ok = rpc:call(Node, emqx_ct_helpers, start_apps, [StartApps, EnvHandler]),
```

Comment:
you don't want to update `SLAVE_START_APPS` macro to include `emqx` and `emqx_modules` by default?

Reply:
we only need to load the modules (but not necessarily start the app) before `ekka:join`